src/leap/mail/imap/account.py

```diff
@@ -88,7 +88,7 @@ def __init__(self, user_id, store, d=defer.Deferred()):
         # about user_id, only the client backend.
 
         self.user_id = user_id
-        self.account = Account(store, ready_cb=lambda: d.callback(self))
+        self.account = Account(store, user_id, ready_cb=lambda: d.callback(self))
```

Comment:
minor change: do you mind to change the order of the parameters, so that it matches the signature for IMAPAccount?